runtime/vm/resolvesupport.cpp

```diff
@@ -1979,6 +2100,59 @@ resolveMethodHandleRef(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA cpInde
 
 }
 
+j9object_t
+resolveMethodHandle(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA cpIndex, UDATA resolveFlags)
+{
+	J9JavaVM *vm = vmThread->javaVM;
+	j9object_t memberName = NULL;
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	bool canRunJavaCode = J9_ARE_NO_BITS_SET(resolveFlags, J9_RESOLVE_FLAG_JIT_COMPILE_TIME | J9_RESOLVE_FLAG_REDEFINE_CLASS);
+
+	if (!canRunJavaCode) {
+		return NULL;
```

Comment:
Should this check for already resolved before returning null?  I wouldn't expect the resolution to happen if it hasn't but shouldn't we let the JIT see already resolved MHs?